MaterialComponentsBeta.podspec

```diff
@@ -104,6 +106,16 @@ Pod::Spec.new do |mdc|
     end
   end
 
+  mdc.subspec "TextFields+Beta" do |component|
+    component.ios.deployment_target = '9.0'
+    component.public_header_files = MixedReadinessComponents.text_fields.beta_public_header_files
+    component.source_files = MixedReadinessComponents.text_fields.beta_source_files
+
+    component.test_spec 'UnitTests' do |unit_tests|
+      unit_tests.source_files = MixedReadinessComponents.text_fields.beta_unit_test_source_files
```

Comment:
The podspec file's targets are currently configurable at the point of definition. This change introduces the need to move between files to update the configuration, which introduces an extra step + file to manage. Can we consolidate the configuration back to this file while still enabling us to create beta targets?

Reply:
The aim behind this change was actually to simplify the management of a mixed readiness component from the Cocoapods side. With this approach the list of beta file paths is created and updated in one place that is referred to by both podspecs, instead of in both podspecs. In the beta podspec it's returned as is. In the non-beta podspec it's subtracted from the default set of file paths. Are you saying you would rather just manually maintain the list of beta file paths in both podspecs? As the regular source paths in the beta podspec and as exclude paths in the non-beta podspec?

Reply:
I see; thank you for the explanation. Please add this explanation to the PR description.

Reply:
Added documentation in 9cd488d